cmd/agent/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Build the inference-agent binary
```

Comment:
This dockerfile can be removed since we already have a dockerfile for agent here: https://github.com/kubeflow/kfserving/blob/master/agent.Dockerfile

Reply:
Fixed this along with your other comment during my most recent rebase.